compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -296,6 +336,40 @@ namespace
  *         |
  *         V
  *       [Out]
+ *-------------------------------------------------------------------
+ * Version_6 (For only 3D I/O)
```

Comment:
if the pattern is same as existing one, can you please add that info, like;

```diff
- * Version_6 (For only 3D I/O)
+ * Version_6 (Same as Version_2, For only 3D I/O)
```

Reply:
```diff
- * Version_6 (For only 3D I/O)
+ * Version_6 (Same as Version_5, For only 3D I/O)
```